netdata/netdata
# Remove backends subsystem

##### Summary
SSIA

Closes #7017

##### Test Plan
1. Check if `http://localhost:19999/netdata.conf` does not contain the `backend` section.
2. Check if `http://arch-esxi:19999/api/v1/allmetrics?format=prometheus` is working correctly.
3. Check if exporting connectors are working.
